### DISCUSSION TITLE
Connect disks

### DISCUSSION THREAD

**Discussion by RanchoHam** [udo-munk/RP2040-GEEK-80#57] [Q&A] [answered]
The disks/readme states `...When you run the picosim machine and you want to boot an OS,
put a bootable disk image into drive 0. ...`  
How?  
In z80pack, I would link `ln disks/library/cpm22.dsk disks/drivea.dsk`; but I'm am unsure of how to accomplish this through the FAT32 partition containing DISKS80.

**Comment by udo-munk** [comment, marked as answer]
Don't try that, this is different. Plug it into some computer and connect with a terminal program to the USB serial port. Putty, minicom, cu, screen .... whatever you like. It has a configuration menu and there it will let you mount/unmount disks in the drives.

**Reply by RanchoHam** [reply]
OK, that worked out.  I notice that the f command lists files in the CODE80 directory.  Any way to list the files in the DISKS80 directory?

**Reply by RanchoHam** [reply]
Wow, after I ran it and rebooted it; I see a command to list the disks directory files.  Super!  Case closed!

**Comment by udo-munk** [comment]
Excellent. I hope the menu is explanatory enough and we don't have to write lots of documentation for this. Anyway, in case of problems with it please ask, then we know what needs to be documented because not obvious enough.
